src/shapes/object.class.ts

```diff
@@ -625,6 +618,8 @@ export class FabricObject extends ObjectGeometry {
    */
   static __uid = 0;
 
+  callSuper?: TCallSuper;
+
```

Comment:
this is a compatibility layer for classes that now import extended classes that are not created with fabric.createClass